deployability/modules/workflow_engine/examples/central_components/aws/dtt1-central_components-poc-vagrant.yaml

```diff
@@ -0,0 +1,69 @@
+version: 0.1
```

Comment:
The name of this file must be dtt1-central_components-poc-aws.yaml

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/5289/commits/8d3aa5d0be7c41bff8d5f6966ad7ba61b1bf8d6f